scripts/bootstrap.sh

```diff
@@ -12,12 +12,12 @@ fi
 source "$VIRTUAL_ENV/bin/activate"
 
 # install dev + prod dependencies
-${VIRTUAL_ENV}/bin/pip install tools/peep-2.*tar.gz
-${VIRTUAL_ENV}/bin/peep install --download-cache=./pip-cache -r requirements.txt
+${VIRTUAL_ENV}/bin/python tools/pipstrap.py
+${VIRTUAL_ENV}/bin/pip install --require-hashes -r requirements.txt
 
 # only install `linux-requirements.txt` if you're on linux
 if [[ `uname` == 'Linux' ]]; then
-  ${VIRTUAL_ENV}/bin/peep install --download-cache=./pip-cache -r linux-requirements.txt
+  ${VIRTUAL_ENV}/bin/pip install --require-hashes -r linux-requirements.txt
```

Comment:
[Environment markers](https://www.python.org/dev/peps/pep-0496/) might let you avoid the branch here. They're supported in reqs files from pip 6 onward.